source/wp-content/themes/wporg-developer-2023/theme.json

```diff
@@ -244,12 +257,30 @@
 		},
 		"elements": {
 			"h2": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--heading-4)"
+				},
 				"spacing": {
 					"margin": {
 						"top": "0",
 						"bottom": "var(--wp--style--block-gap)"
 					}
 				}
+			},
+			"h3": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--heading-5)"
+				}
+			},
+			"h4": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--heading-6)"
+				}
+			},
+			"h5": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--heading-6)"
+				}
```

Comment:
I saw "h6" is used in https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/src/code-title/edit.js#L38-L39 & https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/embed.php#L155, should we also give it the custom style?

Reply:
The `h6`s should already have `"fontSize": "var(--wp--preset--font-size--heading-6)"` inherited from the parent theme, so we don't need to set it here.

Reply:
Ah, I see. Thanks for clarifying!